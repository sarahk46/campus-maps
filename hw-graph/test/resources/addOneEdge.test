# Tests that a graph can be created and a pair of nodes
# and an edge between them can be added.

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

#Add one edge
AddEdge graph1 n1 n2 e1

