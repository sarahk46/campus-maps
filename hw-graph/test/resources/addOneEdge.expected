# Tests that a graph can be created and a pair of nodes
# and an edge between them can be added.

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

#Add one edge
added edge e1 from n1 to n2 in graph1

